dirty_cat/target_encoder.py

```diff
@@ -151,7 +161,7 @@ def fit(self, X, y):
                 le.classes_ = np.array(self.categories[j])
 
         self.categories_ = [le.classes_ for le in self._label_encoders_]
-        self.n = len(y)
+        self._n = len(y)
```

Comment:
I think that typically we would use "n_", according to scikit-learn conventions.